### PR TITLE
ActionView unit tests

### PR DESCRIPTION
**Pull request by fegloff**

### CODE REVIEW

File: voice/voice-ai/x/Actions/ActionsView.swift [hunk 1]
```diff
@@ -468,8 +468,8 @@ struct ActionsView: View {
     }
 }
 
-#Preview {
-    NavigationView {
-        ActionsView()
-    }
-}
+//#Preview {
```
Comment (nageshkumarmishra) [inline voice/voice-ai/x/Actions/ActionsView.swift:471]: Why is this code commented on?

**Comment by nageshkumarmishra** [comment]
I have fixed the conflicts.

**Review by nageshkumarmishra — APPROVED** [review]

File: voice/voice-ai/x/IAP/Store.swift [hunk 2]
```diff
@@ -29,6 +29,7 @@ class Store: ObservableObject {
             print("[Store] Products:", products)
         } catch {
             print(error)
+            products = []
```
Comment (polymorpher) [inline voice/voice-ai/x/IAP/Store.swift:32]: should catch errors with Sentry here as well

**Comment by polymorpher** [comment]
Tests seem to be a bit shallow. Should it not check the state of ActionView, state of related classes, and various actions? (long press, drag, stop)

**Pull request merged**